### PR TITLE
add 64 busybox

### LINKED ISSUE #656: chown: Value too large for defined data type
Kaniko build fails (randomly) with `Value too large for defined data type`

Dockerfile snippet:

```
FROM <repo>ubuntu:latest
WORKDIR /opt/docker
ADD opt /opt
RUN ["chown", "-R", "test_nobody:test_nobody", "."]
```

Build error :

```
INFO[0021] Taking snapshot of files...                  
INFO[0023] RUN ["chown", "-R", "test_nobody:test_nobody", "."] 
INFO[0023] cmd: chown                                   
INFO[0023] args: [-R test_nobody:test_nobody .]         
  chown: .: Value too large for defined data type
  error building image: error building stage: waiting for process to exit: exit status 1
```

Kaniko : gcr.io/kaniko-project/executor:debug
Build Env: Jenkins - Kubernetes

### PR DESCRIPTION
**Pull request by tejal29**
Trying to 
Fixes #656

Copying the /busybox/busybox from amb46/busybox to see if this issue is solved.

### CODE REVIEW

**Comment by tejal29** [comment]
TODO run container diff on tis debug image and `gcr.io/kaniko-project/executor:debug`

File: deploy/Dockerfile_debug [hunk 1]
```diff
@@ -46,6 +46,9 @@ COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
 COPY --from=0 /go/src/github.com/awslabs/amazon-ecr-credential-helper/bin/linux-amd64/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
 COPY --from=0 /usr/local/bin/docker-credential-acr-linux /kaniko/docker-credential-acr
 COPY --from=1 /distroless/bazel-bin/experimental/busybox/busybox/ /busybox/
+# Workaround See #https://github.com/GoogleContainerTools/kaniko/issues/656#issuecomment-564035645
+COPY --from=amd64/busybox:1.31.0 /bin/busybox /busybox/busybox
```
Comment (den-is) [inline deploy/Dockerfile_debug:50]: why not **1.31.1** ?
https://hub.docker.com/_/busybox
https://busybox.net/

Comment (tejal29) [reply deploy/Dockerfile_debug:50]: i can definitely pick up the latest version. 
Will update it. Thanks for bringing that to my attention.

**Comment by alex1989hu** [comment]
Related https://github.com/GoogleContainerTools/kaniko/issues/1277

File: deploy/Dockerfile_debug [hunk 2]
```diff
@@ -46,6 +46,9 @@ COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
 COPY --from=0 /go/src/github.com/awslabs/amazon-ecr-credential-helper/bin/linux-amd64/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
 COPY --from=0 /usr/local/bin/docker-credential-acr-linux /kaniko/docker-credential-acr
 COPY --from=1 /distroless/bazel-bin/experimental/busybox/busybox/ /busybox/
```
Comment (alex1989hu) [inline deploy/Dockerfile_debug:48]: Based on https://github.com/GoogleContainerTools/kaniko/issues/1277 - distroless provided busybox does not add any extra. So the `# Workaround comment` can be removed.

File: deploy/Dockerfile_debug [hunk 3]
```diff
@@ -33,21 +33,17 @@ RUN mkdir -p /kaniko/.docker
...
-WORKDIR /distroless
-RUN bazel build //experimental/busybox:busybox_tar
-RUN tar -C /distroless/bazel-bin/experimental/busybox/ -xf /distroless/bazel-bin/experimental/busybox/busybox.tar
-
 FROM scratch
 COPY --from=0 /go/src/github.com/GoogleContainerTools/kaniko/out/* /kaniko/
 COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
 COPY --from=0 /go/src/github.com/awslabs/amazon-ecr-credential-helper/bin/linux-amd64/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
 COPY --from=0 /usr/local/bin/docker-credential-acr-linux /kaniko/docker-credential-acr
 COPY --from=1 /distroless/bazel-bin/experimental/busybox/busybox/ /busybox/
```
Comment (alex1989hu) [inline deploy/Dockerfile_debug:41]: This is no more needed

**Comment by tejal29** [comment]
@alex1989hu Thans you so much for running the container-diff.
The diff is
```
[
  {
    "Image1": "gcr.io/kaniko-project/executor:debug",
    "Image2": "gcr.io/kaniko-project/executor:debug-v0.22.0",
    "DiffType": "File",
    "Diff": {
      "Adds": [
        {
          "Name": "/busybox/[",
          "Size": 979472
        },
        {
          "Name": "/busybox/[[",
          "Size": 979472
        },
        {
          "Name": "/busybox/acpid",
          "Size": 979472
        },
        {
          "Name": "/busybox/add-shell",
          "Size": 979472
        },
        {
          "Name": "/busybox/addgroup",
          "Size": 979472
        },
        {
          "Name": "/busybox/adduser",
          "Size": 979472
        },
        {
          "Name": "/busybox/adjtimex",
          "Size": 979472
        },
        {
          "Name": "/busybox/ar",
          "Size": 979472
        },
        {
          "Name": "/busybox/arp",
          "Size": 979472
        },
        {
          "Name": "/busybox/arping",
          "Size": 979472
        },
        {
          "Name": "/busybox/ash",
          "Size": 979472
        },
        {
          "Name": "/busybox/awk",
          "Size": 979472
        },
        {
          "Name": "/busybox/base64",
          "Size": 979472
        },
        {
          "Name": "/busybox/basename",
          "Size": 979472
        },
        {
          "Name": "/busybox/blkdiscard",
          "Size": 979472
        },
        {
          "Name": "/busybox/blkid",
          "Size": 979472
        },
        {
          "Name": "/busybox/blockdev",
          "Size": 979472
        },
        {
          "Name": "/busybox/bootchartd",
          "Size": 979472
        },
        {
          "Name": "/busybox/brctl",
          "Size": 979472
        },
        {
          "Name": "/busybox/bunzip2",
          "Size": 979472
        },
        {
          "Name": "/busybox/bzcat",
          "Size": 979472
        },
        {
          "Name": "/busybox/bzip2",
          "Size": 979472
        },
        {
          "Name": "/busybox/cal",
          "Size": 979472
        },
        {
          "Name": "/busybox/cat",
          "Size": 979472
        },
        {
          "Name": "/busybox/chat",
          "Size": 979472
        },
        {
          "Name": "/busybox/chattr",
          "Size": 979472
        },
        {
          "Name": "/busybox/chgrp",
          "Size": 979472
        },
        {
          "Name": "/busybox/chmod",
          "Size": 979472
        },
        {
          "Name": "/busybox/chown",
          "Size": 979472
        },
        {
          "Name": "/busybox/chpasswd",
          "Size": 979472
        },
        {
          "Name": "/busybox/chpst",
          "Size": 979472
        },
        {
          "Name": "/busybox/chroot",
          "Size": 979472
        },
        {
          "Name": "/busybox/chrt",
          "Size": 979472
        },
        {
          "Name": "/busybox/chvt",
          "Size": 979472
        },
        {
          "Name": "/busybox/cksum",
          "Size": 979472
        },
        {
          "Name": "/busybox/clear",
          "Size": 979472
        },
        {
          "Name": "/busybox/cmp",
          "Size": 979472
        },
        {
          "Name": "/busybox/comm",
          "Size": 979472
        },
        {
          "Name": "/busybox/conspy",
          "Size": 979472
        },
        {
          "Name": "/busybox/cp",
          "Size": 979472
        },
        {
          "Name": "/busybox/cpio",
          "Size": 979472
        },
        {
          "Name": "/busybox/crond",
          "Size": 979472
        },
        {
          "Name": "/busybox/crontab",
          "Size": 979472
        },
        {
          "Name": "/busybox/cryptpw",
          "Size": 979472
        },
        {
          "Name": "/busybox/cttyhack",
          "Size": 979472
        },
        {
          "Name": "/busybox/cut",
          "Size": 979472
        },
        {
          "Name": "/busybox/date",
          "Size": 979472
        },
        {
          "Name": "/busybox/dc",
          "Size": 979472
        },
        {
          "Name": "/busybox/dd",
          "Size": 979472
        },
        {
          "Name": "/busybox/deallocvt",
          "Size": 979472
        },
        {
          "Name": "/busybox/delgroup",
          "Size": 979472
        },
        {
          "Name": "/busybox/deluser",
          "Size": 979472
        },
        {
          "Name": "/busybox/depmod",
          "Size": 979472
        },
        {
          "Name": "/busybox/devmem",
          "Size": 979472
        },
        {
          "Name": "/busybox/df",
          "Size": 979472
        },
        {
          "Name": "/busybox/dhcprelay",
          "Size": 979472
        },
        {
          "Name": "/busybox/diff",
          "Size": 979472
        },
        {
          "Name": "/busybox/dirname",
          "Size": 979472
        },
        {
          "Name": "/busybox/dmesg",
          "Size": 979472
        },
        {
          "Name": "/busybox/dnsd",
          "Size": 979472
        },
        {
          "Name": "/busybox/dnsdomainname",
          "Size": 979472
        },
        {
          "Name": "/busybox/dos2unix",
          "Size": 979472
        },
        {
          "Name": "/busybox/dpkg",
          "Size": 979472
        },
        {
          "Name": "/busybox/dpkg-deb",
          "Size": 979472
        },
        {
          "Name": "/busybox/du",
          "Size": 979472
        },
        {
          "Name": "/busybox/dumpkmap",
          "Size": 979472
        },
        {
          "Name": "/busybox/dumpleases",
          "Size": 979472
        },
        {
          "Name": "/busybox/echo",
          "Size": 979472
        },
        {
          "Name": "/busybox/ed",
          "Size": 979472
        },
        {
          "Name": "/busybox/egrep",
          "Size": 979472
        },
        {
          "Name": "/busybox/eject",
          "Size": 979472
        },
        {
          "Name": "/busybox/env",
          "Size": 979472
        },
        {
          "Name": "/busybox/envdir",
          "Size": 979472
        },
        {
          "Name": "/busybox/envuidgid",
          "Size": 979472
        },
        {
          "Name": "/busybox/expand",
          "Size": 979472
        },
        {
          "Name": "/busybox/expr",
          "Size": 979472
        },
        {
          "Name": "/busybox/factor",
          "Size": 979472
        },
        {
          "Name": "/busybox/fakeidentd",
          "Size": 979472
        },
        {
          "Name": "/busybox/false",
          "Size": 979472
        },
        {
          "Name": "/busybox/fatattr",
          "Size": 979472
        },
        {
          "Name": "/busybox/fbset",
          "Size": 979472
        },
        {
          "Name": "/busybox/fbsplash",
          "Size": 979472
        },
        {
          "Name": "/busybox/fdflush",
          "Size": 979472
        },
        {
          "Name": "/busybox/fdformat",
          "Size": 979472
        },
        {
          "Name": "/busybox/fdisk",
          "Size": 979472
        },
        {
          "Name": "/busybox/fgconsole",
          "Size": 979472
        },
        {
          "Name": "/busybox/fgrep",
          "Size": 979472
        },
        {
          "Name": "/busybox/find",
          "Size": 979472
        },
        {
          "Name": "/busybox/findfs",
          "Size": 979472
        },
        {
          "Name": "/busybox/flash_eraseall",
          "Size": 979472
        },
        {
          "Name": "/busybox/flash_lock",
          "Size": 979472
        },
        {
          "Name": "/busybox/flash_unlock",
          "Size": 979472
        },
        {
          "Name": "/busybox/flashcp",
          "Size": 979472
        },
        {
          "Name": "/busybox/flock",
          "Size": 979472
        },
        {
          "Name": "/busybox/fold",
          "Size": 979472
        },
        {
          "Name": "/busybox/free",
          "Size": 979472
        },
        {
          "Name": "/busybox/freeramdisk",
          "Size": 979472
        },
        {
          "Name": "/busybox/fsck",
          "Size": 979472
        },
        {
          "Name": "/busybox/fsck.minix",
          "Size": 979472
        },
        {
          "Name": "/busybox/fsfreeze",
          "Size": 979472
        },
        {
          "Name": "/busybox/fstrim",
          "Size": 979472
        },
        {
          "Name": "/busybox/fsync",
          "Size": 979472
        },
        {
          "Name": "/busybox/ftpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/ftpget",
          "Size": 979472
        },
        {
          "Name": "/busybox/ftpput",
          "Size": 979472
        },
        {
          "Name": "/busybox/fuser",
          "Size": 979472
        },
        {
          "Name": "/busybox/getopt",
          "Size": 979472
        },
        {
          "Name": "/busybox/getty",
          "Size": 979472
        },
        {
          "Name": "/busybox/grep",
          "Size": 979472
        },
        {
          "Name": "/busybox/groups",
          "Size": 979472
        },
        {
          "Name": "/busybox/gunzip",
          "Size": 979472
        },
        {
          "Name": "/busybox/gzip",
          "Size": 979472
        },
        {
          "Name": "/busybox/halt",
          "Size": 979472
        },
        {
          "Name": "/busybox/hd",
          "Size": 979472
        },
        {
          "Name": "/busybox/hdparm",
          "Size": 979472
        },
        {
          "Name": "/busybox/head",
          "Size": 979472
        },
        {
          "Name": "/busybox/hexdump",
          "Size": 979472
        },
        {
          "Name": "/busybox/hostid",
          "Size": 979472
        },
        {
          "Name": "/busybox/hostname",
          "Size": 979472
        },
        {
          "Name": "/busybox/httpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/hush",
          "Size": 979472
        },
        {
          "Name": "/busybox/hwclock",
          "Size": 979472
        },
        {
          "Name": "/busybox/i2cdetect",
          "Size": 979472
        },
        {
          "Name": "/busybox/i2cdump",
          "Size": 979472
        },
        {
          "Name": "/busybox/i2cget",
          "Size": 979472
        },
        {
          "Name": "/busybox/i2cset",
          "Size": 979472
        },
        {
          "Name": "/busybox/id",
          "Size": 979472
        },
        {
          "Name": "/busybox/ifconfig",
          "Size": 979472
        },
        {
          "Name": "/busybox/ifenslave",
          "Size": 979472
        },
        {
          "Name": "/busybox/ifplugd",
          "Size": 979472
        },
        {
          "Name": "/busybox/inetd",
          "Size": 979472
        },
        {
          "Name": "/busybox/init",
          "Size": 979472
        },
        {
          "Name": "/busybox/inotifyd",
          "Size": 979472
        },
        {
          "Name": "/busybox/insmod",
          "Size": 979472
        },
        {
          "Name": "/busybox/install",
          "Size": 979472
        },
        {
          "Name": "/busybox/ionice",
          "Size": 979472
        },
        {
          "Name": "/busybox/iostat",
          "Size": 979472
        },
        {
          "Name": "/busybox/ip",
          "Size": 979472
        },
        {
          "Name": "/busybox/ipaddr",
          "Size": 979472
        },
        {
          "Name": "/busybox/ipcalc",
          "Size": 979472
        },
        {
          "Name": "/busybox/ipcrm",
          "Size": 979472
        },
        {
          "Name": "/busybox/ipcs",
          "Size": 979472
        },
        {
          "Name": "/busybox/iplink",
          "Size": 979472
        },
        {
          "Name": "/busybox/ipneigh",
          "Size": 979472
        },
        {
          "Name": "/busybox/iproute",
          "Size": 979472
        },
        {
          "Name": "/busybox/iprule",
          "Size": 979472
        },
        {
          "Name": "/busybox/iptunnel",
          "Size": 979472
        },
        {
          "Name": "/busybox/kbd_mode",
          "Size": 979472
        },
        {
          "Name": "/busybox/kill",
          "Size": 979472
        },
        {
          "Name": "/busybox/killall",
          "Size": 979472
        },
        {
          "Name": "/busybox/killall5",
          "Size": 979472
        },
        {
          "Name": "/busybox/klogd",
          "Size": 979472
        },
        {
          "Name": "/busybox/last",
          "Size": 979472
        },
        {
          "Name": "/busybox/less",
          "Size": 979472
        },
        {
          "Name": "/busybox/link",
          "Size": 979472
        },
        {
          "Name": "/busybox/linux32",
          "Size": 979472
        },
        {
          "Name": "/busybox/linux64",
          "Size": 979472
        },
        {
          "Name": "/busybox/linuxrc",
          "Size": 979472
        },
        {
          "Name": "/busybox/ln",
          "Size": 979472
        },
        {
          "Name": "/busybox/loadfont",
          "Size": 979472
        },
        {
          "Name": "/busybox/loadkmap",
          "Size": 979472
        },
        {
          "Name": "/busybox/logger",
          "Size": 979472
        },
        {
          "Name": "/busybox/login",
          "Size": 979472
        },
        {
          "Name": "/busybox/logname",
          "Size": 979472
        },
        {
          "Name": "/busybox/losetup",
          "Size": 979472
        },
        {
          "Name": "/busybox/lpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/lpq",
          "Size": 979472
        },
        {
          "Name": "/busybox/lpr",
          "Size": 979472
        },
        {
          "Name": "/busybox/ls",
          "Size": 979472
        },
        {
          "Name": "/busybox/lsattr",
          "Size": 979472
        },
        {
          "Name": "/busybox/lsmod",
          "Size": 979472
        },
        {
          "Name": "/busybox/lsof",
          "Size": 979472
        },
        {
          "Name": "/busybox/lspci",
          "Size": 979472
        },
        {
          "Name": "/busybox/lsscsi",
          "Size": 979472
        },
        {
          "Name": "/busybox/lsusb",
          "Size": 979472
        },
        {
          "Name": "/busybox/lzcat",
          "Size": 979472
        },
        {
          "Name": "/busybox/lzma",
          "Size": 979472
        },
        {
          "Name": "/busybox/lzop",
          "Size": 979472
        },
        {
          "Name": "/busybox/lzopcat",
          "Size": 979472
        },
        {
          "Name": "/busybox/makedevs",
          "Size": 979472
        },
        {
          "Name": "/busybox/makemime",
          "Size": 979472
        },
        {
          "Name": "/busybox/man",
          "Size": 979472
        },
        {
          "Name": "/busybox/md5sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/mdev",
          "Size": 979472
        },
        {
          "Name": "/busybox/mesg",
          "Size": 979472
        },
        {
          "Name": "/busybox/microcom",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkdir",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkdosfs",
          "Size": 979472
        },
        {
          "Name": "/busybox/mke2fs",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkfifo",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkfs.ext2",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkfs.minix",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkfs.vfat",
          "Size": 979472
        },
        {
          "Name": "/busybox/mknod",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkpasswd",
          "Size": 979472
        },
        {
          "Name": "/busybox/mkswap",
          "Size": 979472
        },
        {
          "Name": "/busybox/mktemp",
          "Size": 979472
        },
        {
          "Name": "/busybox/modinfo",
          "Size": 979472
        },
        {
          "Name": "/busybox/modprobe",
          "Size": 979472
        },
        {
          "Name": "/busybox/more",
          "Size": 979472
        },
        {
          "Name": "/busybox/mount",
          "Size": 979472
        },
        {
          "Name": "/busybox/mountpoint",
          "Size": 979472
        },
        {
          "Name": "/busybox/mpstat",
          "Size": 979472
        },
        {
          "Name": "/busybox/mt",
          "Size": 979472
        },
        {
          "Name": "/busybox/mv",
          "Size": 979472
        },
        {
          "Name": "/busybox/nameif",
          "Size": 979472
        },
        {
          "Name": "/busybox/nbd-client",
          "Size": 979472
        },
        {
          "Name": "/busybox/nc",
          "Size": 979472
        },
        {
          "Name": "/busybox/netstat",
          "Size": 979472
        },
        {
          "Name": "/busybox/nice",
          "Size": 979472
        },
        {
          "Name": "/busybox/nl",
          "Size": 979472
        },
        {
          "Name": "/busybox/nmeter",
          "Size": 979472
        },
        {
          "Name": "/busybox/nohup",
          "Size": 979472
        },
        {
          "Name": "/busybox/nproc",
          "Size": 979472
        },
        {
          "Name": "/busybox/ntpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/od",
          "Size": 979472
        },
        {
          "Name": "/busybox/openvt",
          "Size": 979472
        },
        {
          "Name": "/busybox/partprobe",
          "Size": 979472
        },
        {
          "Name": "/busybox/passwd",
          "Size": 979472
        },
        {
          "Name": "/busybox/paste",
          "Size": 979472
        },
        {
          "Name": "/busybox/patch",
          "Size": 979472
        },
        {
          "Name": "/busybox/pgrep",
          "Size": 979472
        },
        {
          "Name": "/busybox/pidof",
          "Size": 979472
        },
        {
          "Name": "/busybox/ping",
          "Size": 979472
        },
        {
          "Name": "/busybox/ping6",
          "Size": 979472
        },
        {
          "Name": "/busybox/pipe_progress",
          "Size": 979472
        },
        {
          "Name": "/busybox/pivot_root",
          "Size": 979472
        },
        {
          "Name": "/busybox/pkill",
          "Size": 979472
        },
        {
          "Name": "/busybox/pmap",
          "Size": 979472
        },
        {
          "Name": "/busybox/popmaildir",
          "Size": 979472
        },
        {
          "Name": "/busybox/poweroff",
          "Size": 979472
        },
        {
          "Name": "/busybox/powertop",
          "Size": 979472
        },
        {
          "Name": "/busybox/printenv",
          "Size": 979472
        },
        {
          "Name": "/busybox/printf",
          "Size": 979472
        },
        {
          "Name": "/busybox/ps",
          "Size": 979472
        },
        {
          "Name": "/busybox/pscan",
          "Size": 979472
        },
        {
          "Name": "/busybox/pstree",
          "Size": 979472
        },
        {
          "Name": "/busybox/pwd",
          "Size": 979472
        },
        {
          "Name": "/busybox/pwdx",
          "Size": 979472
        },
        {
          "Name": "/busybox/raidautorun",
          "Size": 979472
        },
        {
          "Name": "/busybox/rdate",
          "Size": 979472
        },
        {
          "Name": "/busybox/rdev",
          "Size": 979472
        },
        {
          "Name": "/busybox/readlink",
          "Size": 979472
        },
        {
          "Name": "/busybox/readprofile",
          "Size": 979472
        },
        {
          "Name": "/busybox/realpath",
          "Size": 979472
        },
        {
          "Name": "/busybox/reboot",
          "Size": 979472
        },
        {
          "Name": "/busybox/reformime",
          "Size": 979472
        },
        {
          "Name": "/busybox/remove-shell",
          "Size": 979472
        },
        {
          "Name": "/busybox/renice",
          "Size": 979472
        },
        {
          "Name": "/busybox/reset",
          "Size": 979472
        },
        {
          "Name": "/busybox/resize",
          "Size": 979472
        },
        {
          "Name": "/busybox/rev",
          "Size": 979472
        },
        {
          "Name": "/busybox/rm",
          "Size": 979472
        },
        {
          "Name": "/busybox/rmdir",
          "Size": 979472
        },
        {
          "Name": "/busybox/rmmod",
          "Size": 979472
        },
        {
          "Name": "/busybox/route",
          "Size": 979472
        },
        {
          "Name": "/busybox/rpm",
          "Size": 979472
        },
        {
          "Name": "/busybox/rpm2cpio",
          "Size": 979472
        },
        {
          "Name": "/busybox/rtcwake",
          "Size": 979472
        },
        {
          "Name": "/busybox/run-parts",
          "Size": 979472
        },
        {
          "Name": "/busybox/runlevel",
          "Size": 979472
        },
        {
          "Name": "/busybox/runsv",
          "Size": 979472
        },
        {
          "Name": "/busybox/runsvdir",
          "Size": 979472
        },
        {
          "Name": "/busybox/rx",
          "Size": 979472
        },
        {
          "Name": "/busybox/script",
          "Size": 979472
        },
        {
          "Name": "/busybox/scriptreplay",
          "Size": 979472
        },
        {
          "Name": "/busybox/sed",
          "Size": 979472
        },
        {
          "Name": "/busybox/sendmail",
          "Size": 979472
        },
        {
          "Name": "/busybox/seq",
          "Size": 979472
        },
        {
          "Name": "/busybox/setarch",
          "Size": 979472
        },
        {
          "Name": "/busybox/setconsole",
          "Size": 979472
        },
        {
          "Name": "/busybox/setfont",
          "Size": 979472
        },
        {
          "Name": "/busybox/setkeycodes",
          "Size": 979472
        },
        {
          "Name": "/busybox/setlogcons",
          "Size": 979472
        },
        {
          "Name": "/busybox/setpriv",
          "Size": 979472
        },
        {
          "Name": "/busybox/setserial",
          "Size": 979472
        },
        {
          "Name": "/busybox/setsid",
          "Size": 979472
        },
        {
          "Name": "/busybox/setuidgid",
          "Size": 979472
        },
        {
          "Name": "/busybox/sh",
          "Size": 979472
        },
        {
          "Name": "/busybox/sha1sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/sha256sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/sha3sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/sha512sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/showkey",
          "Size": 979472
        },
        {
          "Name": "/busybox/shred",
          "Size": 979472
        },
        {
          "Name": "/busybox/shuf",
          "Size": 979472
        },
        {
          "Name": "/busybox/slattach",
          "Size": 979472
        },
        {
          "Name": "/busybox/sleep",
          "Size": 979472
        },
        {
          "Name": "/busybox/smemcap",
          "Size": 979472
        },
        {
          "Name": "/busybox/softlimit",
          "Size": 979472
        },
        {
          "Name": "/busybox/sort",
          "Size": 979472
        },
        {
          "Name": "/busybox/split",
          "Size": 979472
        },
        {
          "Name": "/busybox/ssl_client",
          "Size": 979472
        },
        {
          "Name": "/busybox/start-stop-daemon",
          "Size": 979472
        },
        {
          "Name": "/busybox/stat",
          "Size": 979472
        },
        {
          "Name": "/busybox/strings",
          "Size": 979472
        },
        {
          "Name": "/busybox/stty",
          "Size": 979472
        },
        {
          "Name": "/busybox/su",
          "Size": 979472
        },
        {
          "Name": "/busybox/sulogin",
          "Size": 979472
        },
        {
          "Name": "/busybox/sum",
          "Size": 979472
        },
        {
          "Name": "/busybox/sv",
          "Size": 979472
        },
        {
          "Name": "/busybox/svc",
          "Size": 979472
        },
        {
          "Name": "/busybox/svlogd",
          "Size": 979472
        },
        {
          "Name": "/busybox/swapoff",
          "Size": 979472
        },
        {
          "Name": "/busybox/swapon",
          "Size": 979472
        },
        {
          "Name": "/busybox/switch_root",
          "Size": 979472
        },
        {
          "Name": "/busybox/sync",
          "Size": 979472
        },
        {
          "Name": "/busybox/sysctl",
          "Size": 979472
        },
        {
          "Name": "/busybox/syslogd",
          "Size": 979472
        },
        {
          "Name": "/busybox/tac",
          "Size": 979472
        },
        {
          "Name": "/busybox/tail",
          "Size": 979472
        },
        {
          "Name": "/busybox/tar",
          "Size": 979472
        },
        {
          "Name": "/busybox/taskset",
          "Size": 979472
        },
        {
          "Name": "/busybox/tcpsvd",
          "Size": 979472
        },
        {
          "Name": "/busybox/tee",
          "Size": 979472
        },
        {
          "Name": "/busybox/telnet",
          "Size": 979472
        },
        {
          "Name": "/busybox/telnetd",
          "Size": 979472
        },
        {
          "Name": "/busybox/test",
          "Size": 979472
        },
        {
          "Name": "/busybox/tftp",
          "Size": 979472
        },
        {
          "Name": "/busybox/tftpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/time",
          "Size": 979472
        },
        {
          "Name": "/busybox/timeout",
          "Size": 979472
        },
        {
          "Name": "/busybox/top",
          "Size": 979472
        },
        {
          "Name": "/busybox/touch",
          "Size": 979472
        },
        {
          "Name": "/busybox/tr",
          "Size": 979472
        },
        {
          "Name": "/busybox/traceroute",
          "Size": 979472
        },
        {
          "Name": "/busybox/traceroute6",
          "Size": 979472
        },
        {
          "Name": "/busybox/true",
          "Size": 979472
        },
        {
          "Name": "/busybox/truncate",
          "Size": 979472
        },
        {
          "Name": "/busybox/tty",
          "Size": 979472
        },
        {
          "Name": "/busybox/ttysize",
          "Size": 979472
        },
        {
          "Name": "/busybox/tunctl",
          "Size": 979472
        },
        {
          "Name": "/busybox/tune2fs",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubiattach",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubidetach",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubimkvol",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubirename",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubirmvol",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubirsvol",
          "Size": 979472
        },
        {
          "Name": "/busybox/ubiupdatevol",
          "Size": 979472
        },
        {
          "Name": "/busybox/udhcpc",
          "Size": 979472
        },
        {
          "Name": "/busybox/udhcpd",
          "Size": 979472
        },
        {
          "Name": "/busybox/udpsvd",
          "Size": 979472
        },
        {
          "Name": "/busybox/uevent",
          "Size": 979472
        },
        {
          "Name": "/busybox/umount",
          "Size": 979472
        },
        {
          "Name": "/busybox/uname",
          "Size": 979472
        },
        {
          "Name": "/busybox/uncompress",
          "Size": 979472
        },
        {
          "Name": "/busybox/unexpand",
          "Size": 979472
        },
        {
          "Name": "/busybox/uniq",
          "Size": 979472
        },
        {
          "Name": "/busybox/unix2dos",
          "Size": 979472
        },
        {
          "Name": "/busybox/unlink",
          "Size": 979472
        },
        {
          "Name": "/busybox/unlzma",
          "Size": 979472
        },
        {
          "Name": "/busybox/unlzop",
          "Size": 979472
        },
        {
          "Name": "/busybox/unxz",
          "Size": 979472
        },
        {
          "Name": "/busybox/unzip",
          "Size": 979472
        },
        {
          "Name": "/busybox/uptime",
          "Size": 979472
        },
        {
          "Name": "/busybox/users",
          "Size": 979472
        },
        {
          "Name": "/busybox/usleep",
          "Size": 979472
        },
        {
          "Name": "/busybox/uudecode",
          "Size": 979472
        },
        {
          "Name": "/busybox/uuencode",
          "Size": 979472
        },
        {
          "Name": "/busybox/vconfig",
          "Size": 979472
        },
        {
          "Name": "/busybox/vi",
          "Size": 979472
        },
        {
          "Name": "/busybox/vlock",
          "Size": 979472
        },
        {
          "Name": "/busybox/volname",
          "Size": 979472
        },
        {
          "Name": "/busybox/w",
          "Size": 979472
        },
        {
          "Name": "/busybox/wall",
          "Size": 979472
        },
        {
          "Name": "/busybox/watch",
          "Size": 979472
        },
        {
          "Name": "/busybox/watchdog",
          "Size": 979472
        },
        {
          "Name": "/busybox/wc",
          "Size": 979472
        },
        {
          "Name": "/busybox/wget",
          "Size": 979472
        },
        {
          "Name": "/busybox/which",
          "Size": 979472
        },
        {
          "Name": "/busybox/who",
          "Size": 979472
        },
        {
          "Name": "/busybox/whoami",
          "Size": 979472
        },
        {
          "Name": "/busybox/whois",
          "Size": 979472
        },
        {
          "Name": "/busybox/xargs",
          "Size": 979472
        },
        {
          "Name": "/busybox/xxd",
          "Size": 979472
        },
        {
          "Name": "/busybox/xz",
          "Size": 979472
        },
        {
          "Name": "/busybox/xzcat",
          "Size": 979472
        },
        {
          "Name": "/busybox/yes",
          "Size": 979472
        },
        {
          "Name": "/busybox/zcat",
          "Size": 979472
        },
        {
          "Name": "/busybox/zcip",
          "Size": 979472
        }
      ],
      "Dels": [
        {
          "Name": "/etc",
          "Size": 782
        },
        {
          "Name": "/etc/nsswitch.conf",
          "Size": 782
        }
      ],
      "Mods": [
        {
          "Name": "/kaniko/executor",
          "Size1": 47337472,
          "Size2": 47333376
        },
        {
          "Name": "/kaniko/warmer",
          "Size1": 31150080,
          "Size2": 31117312
        },
        {
          "Name": "/kaniko/docker-credential-ecr-login",
          "Size1": 8470528,
          "Size2": 8470528
        },
        {
          "Name": "/kaniko/docker-credential-gcr",
          "Size1": 6435607,
          "Size2": 5147454
        },
        {
          "Name": "/busybox/busybox",
          "Size1": 1132800,
          "Size2": 979472
        }
      ]
    }
  }
]
```

File: deploy/Dockerfile_debug (hunk 1 above)
Comment (alex1989hu) [reply deploy/Dockerfile_debug]: I think this is solved now 👍

**Comment by tejal29** [comment]
Verified by @alex1989hu  here https://pastebin.com/1z8uWRpk

File: deploy/Dockerfile_debug [hunk 4]
```diff
@@ -33,21 +33,16 @@ RUN mkdir -p /kaniko/.docker
...
-RUN bazel build //experimental/busybox:busybox_tar
-RUN tar -C /distroless/bazel-bin/experimental/busybox/ -xf /distroless/bazel-bin/experimental/busybox/busybox.tar
-
 FROM scratch
 COPY --from=0 /go/src/github.com/GoogleContainerTools/kaniko/out/* /kaniko/
 COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
 COPY --from=0 /go/src/github.com/awslabs/amazon-ecr-credential-helper/bin/linux-amd64/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
 COPY --from=0 /usr/local/bin/docker-credential-acr-linux /kaniko/docker-credential-acr
-COPY --from=1 /distroless/bazel-bin/experimental/busybox/busybox/ /busybox/
+COPY --from=amd64/busybox:1.31.1 /bin /busybox
```
Comment (Andor) [inline deploy/Dockerfile_debug:41]: I think this line should be the first COPY in this stage to achieve a bit more optimized builds.

**Pull request merged**